ratelimit/swim.go

```diff
@@ -78,11 +79,11 @@ func newClusterRateLimiterSwim(s Settings, sw Swarmer, group string) *clusterLim
 	return rl
 }
 
-// Allow returns true if the request calculated across the cluster of
+// AllowContext returns true if the request calculated across the cluster of
 // skippers should be allowed else false. It will share it's own data
 // and use the current cluster information to calculate global rates
-// to decide to allow or not.
-func (c *clusterLimitSwim) Allow(clearText string) bool {
+// to decide to allow or not. With additional context.Context
```

Comment:
"With additional context.Context" is redundant - context is required by this method